service-tests/liberapay.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'liberapay', title: 'Liberapay' });
+module.exports = t;
+
+t.create('Receiving')
+  .get('/Liberapay/receives.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'receives',
+    value: Joi.string().regex(/^[0-9]+(\.[0-9]{2})?[ A-Za-z]{4}\/week/)
+  }));
+
+t.create('Giving')
+  .get('/Liberapay/gives.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'gives',
+    value: Joi.string().regex(/^[0-9]+(\.[0-9]{2})?[ A-Za-z]{4}\/week/)
```

Comment:
It probably would be a good idea to make sure this does not return $0.00. Perhaps `/^[0-9]*[1-9][0-9]*(\.[0-9]{2})?` would do it?

Could you move the regex up top instead of repeating it, like in the [appveyor tests](https://github.com/badges/shields/blob/master/service-tests/appveyor.js)?

Reply:
It's entirely possible to return 0.00 and be correct.

Reply:
Let's find a project where we're confident it will be > 0.